README.md

```diff
@@ -66,15 +67,15 @@ You can find the further information about the following steps in the [docs](doc
 - Configure OPC UA Connector
 - Collect data in Flow Creator and calculate KPIs
 - Create custom data source (new metadata, publish data to new topic)
-- Install and configure OPC UA configurator and application
+- Install and configure OPC UA connector 
 
 ## Usage
 
-When previous steps are configured correctly, data are available in OPC UA Application. Use UA Expert to connect to OPC UA Application at end point `opc.tcp://Ip-Address-of-Edge-Device:48010`. You can also use second IED. Then data is obtained via OPC UA Connector.
+When previous steps are configured correctly, data are available in OPC UA Application. Use UA Expert to connect to OPC UA Application at end point `opc.tcp://Ip-Address-of-Edge-Device:4840`. You can also use second IED. Then data is obtained via OPC UA Connector.
```

Comment:
Are you sure about the port 4840? In the documentation it says that the correct port ist 48010

_A client can connect to Edge OPC UA server at end point opc.tcp://Ip-Address-ofEdge-Device:48010._